tests/reporter/kubernetes.go

```diff
@@ -164,6 +164,7 @@ func (r *KubernetesReporter) dumpNamespaces(duration time.Duration, vmiNamespace
 	r.logNodes(virtCli, nodes)
 	r.logPods(virtCli, pods)
 	r.logVMs(virtCli)
+	r.logVMSnapshot(virtCli)
```

Comment:
it will be great if you can also add it for vmrestore

Reply:
done